### PR TITLE
Partial Speaker update README.md

### PR DESCRIPTION
**Pull request by ghost**
@rkneufeld. this is obvi a wip. As I continue building out my current serverless website (vendorsecurityalliance.com), I'll have a better topic description. Just wanted to throw something up at the moment.

### CODE REVIEW

File: program/Talk - Nicholas Shook - Clojure, AWS & Lambda/README.md [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
+# "Serverless Clojure on AWS" - Nicholas Shook
 
 > This template should be copied to the folder
 > `program/<Name> - <Title>/README.md`.
 >
 > Remove this notice and replace any TBA sections you can. Feel free to leave
 > those you cannot.
 
-**Format**: *TBA - e.g. "Talk, 40 minutes" or "Workshop, 2 hours"*
+**Format**: Workshop, 2 hours
```
Comment (qtrfeast) [inline program/Talk - Nicholas Shook - Clojure, AWS & Lambda/README.md]: I have you down as 40 minutes?

File: program/Talk - Nicholas Shook - Clojure, AWS & Lambda/README.md [hunk 2]
```diff
@@ -1,30 +1,26 @@
...
 ## About The Author
 
-*TBA - Replace with 1-2 paragraphs describing yourself and relevant background
-information.*
+Nicholas Shook is a developer and lawyer from Las Vegas. He has worked most recently as the technical founder of [Stealth Worker](https://www.stealthworker.com) and has created the website for the [Vendor Security Alliance](https://www.vendorsecurityalliance.org) using serverless architecture.
 
 ### Biographic Details
 
  - Time Zone: *TBA - e.g. UTC+1, Central Standard Time, ...*
  - Website: [TBA][website]
```
Comment (qtrfeast) [inline program/Talk - Nicholas Shook - Clojure, AWS & Lambda/README.md]: Can you change TBA in these to "1337lawyers.com" and "@shicholas" respectively?

**Review by qtrfeast — CHANGES_REQUESTED** [review]
Just two small comments, then we can get this merged in.

File: program/Talk - Nicholas Shook - Clojure, AWS & Lambda/README.md (hunk 1 above)
Comment (ghost) [reply program/Talk - Nicholas Shook - Clojure, AWS & Lambda/README.md]: yup, that's what I thought too, but then I thought you wanted me to go in-depth on a workshop. I'll prepare for just a 40 minute presentation.

**Review by qtrfeast — APPROVED** [review]

**Comment by qtrfeast** [comment]
Thanks!

**Pull request merged**